src/components/POLine/Item/ItemForm.js

```diff
@@ -213,6 +218,22 @@ class ItemForm extends Component {
 
     return (
       <Fragment>
+        <Row>
+          <Col
+            xs={6}
+            md={3}
+          >
+            <Field
+              component={Checkbox}
+              fullWidth
+              label={<FormattedMessage id="ui-orders.poLine.package" />}
+              name="isPackage"
+              onChange={(e, value) => this.onChangeField(value, 'checkinItems')}
```

Comment:
please declare this above and use here as a variable. Later it will be easier to make this as a callback